Sex from chi

Moohan opened the pull request:
Opening as draft pending comments in the issue.
Closes #43

080708km reviewed (changes requested):
Delete

davidc92 commented:
Hi James

I'm just getting round to looking at this now. Before I do a review I'd just like to understand the scope for this function a bit more because I'm not entirely convinced this is all that useful? The reason being, I don't know of any dataset that has chi on it that does not also have sex. I also have some questions around how accurate this is - how are people who are intersex or their sex is otherwise unknown accounted for here? 

D

davidc92 reviewed (changes requested):
Just a few minor comments. While you're updating, just in the interest of brevity, can you re-knit the readme.rmd file so that the .md is in sync as part of this PR please? D

Moohan commented on DESCRIPTION:
```diff
@@ -11,7 +11,8 @@ Authors@R: c(
     person("Alice", "Byers", email = "alice.byers@nhs.net", role = "ctb"),
-    person("Alan", "Yeung", email = "alan.yeung@nhs.net", role = "ctb")
+    person("Alan", "Yeung", email = "alan.yeung@nhs.net", role = "ctb"),
+    person("James", "McMahon", email = "j.mcmahon1@nhs.net", role = "ctb")
```
Done

Moohan commented on R/chi.R:
```diff
@@ -151,3 +151,88 @@ chi_pad <- function(x) {
+#' a factor which will have lables \code{'Male'} and \code{'Female')}
+#'
+#' @param chi_number a CHI number or a vector of CHI numbers with \code{character} class.
+#' @param male_value,female_value optionally supply custom values for Male and Female. Note
```
Not done - no space means that that note applies to both of the parameters. Check the doc with `?sex_from_chi` and you'll see what I mean.

Moohan commented on R/chi.R:
```diff
@@ -151,3 +151,88 @@ chi_pad <- function(x) {
+#' @param chi_number a CHI number or a vector of CHI numbers with \code{character} class.
+#' @param male_value,female_value optionally supply custom values for Male and Female. Note
+#' that that these must be of the same class.
+#' @param as_factor logical, optionally return as a factor with lables \code{'Male'}
```
Done

Moohan commented:
Made most of the changes, see comment above.

I updated the `readme.rmd` and re-knitted, I got the same error again about proxy (#55) ... However, the `.md` file still updated and looks ok to me (both in markdown and if you look at it on GitHub for this branch). There were also merge conflicts for some reason which I resolved.

I also just added a short bit to the readme for this function, I'm pretty happy that they're in sync now.
